src/libs/getPolicyDomainName.ts

```diff
@@ -0,0 +1,3 @@
+export default function getPolicyDomainName(policyID: string): string {
+    return `expensify-policy${policyID}.exfy`;
+}
```

Comment:
NAB: but in future PR can you please move this to the existing policy utils file?